### DISCUSSION TITLE
Documentation typo in "Yellow lookup" section

### DISCUSSION THREAD

**Discussion by csilverman** [datenstrom/community#857] [Report a bug] [answered]
Hi all. I noticed what I think is a documentation typo in the [Yellow Lookup section](https://datenstrom.se/yellow/help/api-for-developers#yellow-lookup). In the code example labeled "Layout file for showing image paths", line 6 is:

```<?php $pathInstall = $this->system->get("coreServerInstallDirectory") ?>```

I think that should be `$this->yellow->system->get("coreServerInstallDirectory")`, right? Without it, I get the error "Undefined property: YellowPage::$system". When I add the `yellow`, it works.

- Chris

**Comment by markseuffert** [comment, marked as answer]
Thank you for the bug report. I updated the [help extension](https://github.com/annaesvensson/yellow-help), it contains all content files for the documentation.

Should be fixed now, please try it out.

**Comment by csilverman** [comment]
Thanks!
